packages/interface-ipfs-core/src/miscellaneous/dns.js

```diff
@@ -28,9 +28,9 @@ export function testDns (factory, options) {
 
     after(() => factory.clean())
 
-    it('should non-recursively resolve ipfs.io', async function () {
+    it('should non-recursively resolve ipfs.tech', async function () {
       try {
-        const res = await ipfs.dns('ipfs.io', { recursive: false })
+        const res = await ipfs.dns('ipfs.tech', { recursive: false })
```

Comment:
This seems to have broken CI:

```
ipfs:   1 failing
ipfs:   1) interface-ipfs-core over ipfs-http-client tests against go-ipfs
ipfs:        .dns
ipfs:          should non-recursively resolve ipfs.tech:
ipfs:      AssertionError: expected '/ipfs/QmdcBdTSBLhS1DVVRXfFYHhGpSQaght…' to match /\/ipns\/.+$/
ipfs:       at Context.<anonymous> (file:///home/runner/work/js-ipfs/js-ipfs/packages/interface-ipfs-core/src/miscellaneous/dns.js:36:24)
ipfs:       at runMicrotasks (<anonymous>)
ipfs:       at processTicksAndRejections (node:internal/process/task_queues:96:5)
ipfs: Command failed with exit code 1: mocha test/interface-http-go.js --ui bdd --require source-map-support/register --timeout=60000 --bail
```

Reply:
That is because on `ipfs.tech` we no longer have a DNSLink that points to other DNSLink: we use CNAME pointing at Fleek instead, and get final `/ipfs/cid` directly, which saves us one DNS lookup.

@achingbrain  I've updated the tests to accept both `/ipfs/` and `/ipns/`: ad0ca96e33684b58cbe425dc8128a3da05d2bf57

Reply:
Is there another handy DNSLink we can use to test recursive lookups?

Reply:
No, nothing stable enough to include in CI/tests.

I think at some point Kub oswitched all resolve methods to be recursive by default?

Also, we've deprecated `ipfs dns ipfs.tech` in Kubo, 
and now simply point people at `ipfs resolve /ipns/ipfs.tech` (recursive by default).